src/core/renderController.ts

```diff
@@ -93,9 +91,6 @@ export module Core {
         // Layout
         _componentsNeedingComputeLayout.values().forEach((component: Component) => component.computeLayout());
 
-        // Top level render; Containers will put their children in the toRender queue
-        _componentsNeedingRender.values().forEach((component: Component) => component.render());
```

Comment:
Turns out it's actually really important for when a child of a `ComponentContainer` gets updated and calls `redraw()`. Digging into *why* now...